src/webots/gui/WbGuiApplication.cpp

```diff
@@ -611,13 +611,8 @@ static void setDarkTitlebar(HWND hwnd) {
 #endif  // _WIN32
 
 void WbGuiApplication::udpateStyleSheet() {
-#ifdef __linux__
-  const QString themeToLoad = WbPreferences::instance()->value("General/theme", "webots_night.qss").toString();
-#else
-  const QString themeToLoad = WbPreferences::instance()->value("General/theme", "webots_classic.qss").toString();
-#endif
-
-  QFile qssFile(WbStandardPaths::resourcesPath() + themeToLoad);
+  mThemeLoaded = WbPreferences::instance()->value("General/theme").toString();
+  QFile qssFile(WbStandardPaths::resourcesPath() + mThemeLoaded);
```

Comment:
Is it normal that we never close the `qssFile`?

Reply:
`QFile` object is closed automatically upon its destructor, hence we do not need to call `close()` explicitly.